pkgs/os-specific/linux/kernel/xanmod-kernels.nix

```
@@ -27,7 +27,7 @@ let
 
     structuredExtraConfig = with lib.kernel; {
       # AMD P-state driver
-      X86_AMD_PSTATE = yes;
+      X86_AMD_PSTATE = lib.mkImageMediaOverride yes;
```

This isn't an image media, so just use `mkOverride` directly

```suggestion
      X86_AMD_PSTATE = lib.mkOverride 60 yes;
```